docs/App.NativeModule.md

```diff
@@ -1,106 +1,122 @@
+> :point_right: I'm not sure this file is linked to / referenced anywhere in the tutorial, except for the intro section. I left a note in `App.tests.md` about linking from that file here.
```

Comment:
@roiberlin - Can you go over Roman's fixes and comments and see if you approve. Thanks 🙏

Reply:
@roiberlin - I'm going to remove Roman comments so I'll be able to merge his changes to master. but when you have time please go over them and update master if needed. Thanks!